classes/mender-image-buildinfo.bbclass

```diff
@@ -1,15 +1,13 @@
-inherit image-buildinfo
-
 DEVICE_TYPE ?= "${MACHINE}"
 IMAGE_ID = "${PN}-${DATETIME}"
-IMAGE_BUILDINFO_VARS = "DISTRO DATETIME PN IMAGE_ID DEVICE_TYPE"
 
 buildinfo_mender () {
 cat > ${IMAGE_ROOTFS}${sysconfdir}/build_mender << END
```

Comment:
If we are changing this maybe we  can change the horrible name of this file. But I guess we can do it in separate PR.